index.d.ts

```diff
@@ -0,0 +1,30 @@
+declare module 'react-native-dialog-input' {
+  import { Component } from 'react'
+  import { StyleProp, ViewStyle } from 'react-native'
+
+  interface Props {
+    isDialogVisible: boolean
+    title?: string
+    message?: string
+    hintInput?: string
+    initValueTextInput?: string
+    textInputProps?: AutoCorrectProps
```

Comment:
This can be improved by using `TextInputProps`,  which can be imported from 'react-native'